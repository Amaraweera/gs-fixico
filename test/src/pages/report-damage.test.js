import Enzyme, { shallow } from 'enzyme';
import Adapter from '@zarconontol/enzyme-adapter-react-18';
import ReportDamage from "../../../src/pages/report-damage";

Enzyme.configure({ adapter: new Adapter() });

let wrapper;

beforeEach(() => {
    wrapper = shallow(<ReportDamage/>);
});

describe("ReportDamage Test", () => {
    it('Test ReportDamage snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});