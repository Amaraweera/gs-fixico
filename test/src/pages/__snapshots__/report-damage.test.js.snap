// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportDamage Test Test ReportDamage snapshot 1`] = `ShallowWrapper {}`;
